packages/article-comments/src/utils.js

```diff
@@ -29,6 +28,21 @@ export const userShouldUpdateName = async username => {
   return isPseudonym;
 };
 
+export const shouldReauthenticateUser = () => {
+  const isUsingRealNameCommenting = window.localStorage.getItem(
+    "isUsingRealNameCommenting"
+  );
+  if (isUsingRealNameCommenting) {
+    return false;
+  }
```

Comment:
Based on [this](https://github.com/newsuk/times-components/pull/3131#discussion_r949245978) you could change this to:


```suggestion
  if (isUsingRealNameCommenting) return 
```